layout/MDErgo1-Default-0.kll

```diff
@@ -42,16 +45,16 @@ U"DELETE" : U"DELETE";
 U"BACKSPACE" : U"BACKSPACE";
 U"FUNCTION5" : U"RIGHT";
 U"FUNCTION4" : U"LEFT";
-U"FUNCTION3" : U"LGUI";
-U"BACKTICK" : U"LALT";
+U"FUNCTION3" : U"BACKSLASH";
+U"BACKTICK" : U"BACKTICK";
 U"LGUI" : U"LCTRL";
 U"FUNCTION6" : U"LCK2";
 U"6" : U"6";
 U"7" : U"7";
 U"8" : U"8";
 U"9" : U"9";
 U"0" : U"0";
-U"MINUS" : U"BACKSPACE";
+U"MINUS" : U"MINUS";
```

Comment:
now that I've retrained 🐶 myself to hit backspace/del with my left thumb, I can move `-` to this more accessible spot